lti_tool/utils.py

```diff
@@ -138,44 +138,53 @@ def sync_user_from_launch(lti_launch: LtiLaunch) -> LtiUser:
     return lti_user
 
 
+def _get_ags_props(lti_launch: LtiLaunch) -> dict:
+    ags_claim = lti_launch.ags_claim
+    if ags_claim is None:
+        return {}
+    ags_scope = ags_claim.get("scope", [])
+    return {
+        "lineitems_url": ags_claim.get("lineitems", ""),
+        "can_query_lineitems": AgsScope.QUERY_LINEITEMS in ags_scope,
+        "can_manage_lineitems": AgsScope.MANAGE_LINEITEMS in ags_scope,
+        "can_publish_scores": AgsScope.PUBLISH_SCORES in ags_scope,
+        "can_access_results": AgsScope.ACCESS_RESULTS in ags_scope,
+    }
+
+
 def sync_context_from_launch(lti_launch: LtiLaunch) -> LtiContext:
     context_claim = lti_launch.context_claim
```

Comment:
This line looks like its obsolete at this point. I'm guessing it's ok to remove?

Reply:
Yep, thanks for taking a close look!